src/components/admin/AdminRestrictedMessage.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { useAsyncFn } from "react-use";
+import firebase from "firebase/compat/app";
+
+import {
+  ATTENDEE_INSIDE_URL,
+  ATTENDEE_LANDING_URL,
+  DISABLED_DUE_TO_1324,
+} from "settings";
+
+import { generateUrl } from "utils/url";
+
+import { useSpaceParams } from "hooks/spaces/useSpaceParams";
+import { useUserId } from "hooks/user/useUserId";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+import { SparkleLogo } from "components/atoms/SparkleLogo";
+
+import SHAPE_DENIED from "assets/images/access-forbidden.svg";
```

Comment:
Might make sense to move this over to the ban icon from font awesome? https://fontawesome.com/icons/ban?s=solid

Reply:
🤷 They don't look like anything really. We'll have to ask for new design first, also for the few missing like "login with no world slug" and "login with only world but no space slug" and probably a new landing page for attendee side spaces etc.